docs/quickstart/python.md

```diff
@@ -14,28 +14,34 @@ working example.</p>
 
 ### Prerequisites
 
-* `pip`: version 8 or higher
+Ensure you have `pip` version 8 or higher:
 
-### Install gRPC
+```sh
+$ python -m pip install --upgrade pip
+```
 
-Install gRPC:
+If you cannot upgrade `pip` due to a system-owned installation, you can
+run the example in a virtualenv:
 
 ```sh
-$ pip install grpcio
+$ python -m pip install virtualenv
+$ virtualenv venv
+$ source venv/bin/activate
+$ python -m pip install --upgrade pip
 ```
 
-Or, to install it system wide:
+### Install gRPC
+
+Install gRPC:
 
 ```sh
-$ sudo pip install grpcio
+$ python -m pip install grpcio
```

Comment:
Before we had this:
""
-If you're on Windows, make sure you installed the `pip.exe` component when you
 -installed Python. Invoke as above but with `pip.exe` instead of `pip` (you may
 -also need to invoke from a `cmd.exe` run as administrator):
""
python -m pip works on both windows and unix,  and is the recommended way of upgrading pip on windows:
https://pip.pypa.io/en/stable/installing/#id11

TLDR; unifies windows/unix instructions.